SNKY-PR-SETUP.md

```diff
@@ -0,0 +1,45 @@
+# How to setup SNYK Autorun on each PR
```

Comment:
Let's combine the sync docs into one at `docs/snyk.md` - ideally docs in the root directory are kept slim.

Reply:
Files combined and moved.